Taskfile.yml

```diff
@@ -50,7 +50,13 @@ tasks:
     desc: Run tests
     dir: "{{.SRC_DIR}}"
     cmds:
-      - go test -race -coverprofile=coverage.txt -covermode=atomic -v ./... {{ .CLI_ARGS }}
+      - go test -race -coverprofile=coverage.txt -covermode=atomic -v ./common/... {{ .CLI_ARGS }}
+
+  test-integration:
```

Comment:
I noticed that these 2 are the same, could add `test` as an alias for this task. https://taskfile.dev/usage/#task-aliases

Reply:
The difference is `task test` runs tests in `./src/common/...` and `task test-integration` runs tests in `./src/cmd/...`

These should be separated as `task test` locally tests code and is part of `task ci`, whereas `task test-integration` is intended for pre-deploy integration testing and is intentionally kept out of regular testing